pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: gcloud
-version: 0.4.0
+version: 0.4.1-dev
```

Comment:
Why a dev release? If we want to publish, make it a normal `+..` release (since there were no big changes).

Reply:
Okay...I usually default to -dev unless I'm sure I'm going to publish